### PR TITLE
Fixed the merge conflict, migrated VC to Swift

### PR DESCRIPTION
**Pull request by charleshkang**
I forgot to migrate to Swift first before making all the storyboard changes, so I had to fix some merge conflicts real quick. I added a `.gitattributes` file (http://roadfiresoftware.com/2015/09/automatically-resolving-git-merge-conflicts-in-xcodes-project-pbxproj/) so it shouldn't happen in the future.

### CODE REVIEW

**Comment by charleshkang** [comment]
Alright, had to deal with some annoying issues along the way, but all the VC classes are now in Swift. I need to update them to Swift 3 soon, but that shouldn't be too bad.

After this gets reviewed and approved, I can finish making the changes to the UI on the ui spike branch

File: InterviewFlashCards/AnswerViewController.swift [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+            answerLabel.text = ""
+        }
+    }
+    
+    func setupTempImageViewBounds() {
+        let width: CGFloat = answerImageView.bounds.size.width
+        let height: CGFloat = answerImageView.bounds.size.height
+        let x: CGFloat = answerImageView.bounds.origin.x
+        let y: CGFloat = answerImageView.bounds.origin.y
+        tempAnswerImageView = UIImageView(frame: CGRectMake(x, y, width, height))
```
Comment (vhart) [inline InterviewFlashCards/AnswerViewController.swift]: Can you use the swift `CGRect` initializer

File: InterviewFlashCards/AnswerViewController.swift [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        view.addGestureRecognizer(tapGestureRecognizer)
+        answerImageView.addGestureRecognizer(pinchGesture)
+    }
+    
+    func showPaginationLabel() {
+        paginationLabel.text = "\(index+1)/\(flashCard.answerImages.count)"
+    }
+    
+    func handleLeftSwipe(gesture: UISwipeGestureRecognizer) {
+        if index < flashCard.answerImages.count-1 {
```
Comment (vhart) [inline InterviewFlashCards/AnswerViewController.swift]: can you make this class more swift like? take advantage of `guard` and other swift constructs. This doesn't have to be a literal copy of the objc file in swift.

File: InterviewFlashCards/AnswerViewController.swift [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+            index += 1
+            showPaginationLabel()
+            answerImageView.image = flashCard.answerImages[index] as? UIImage
+        }
+    }
+    
+    func handleRightSwipe(gesture: UISwipeGestureRecognizer) {
+        if index > 0 {
+            index -= 1
```
Comment (vhart) [inline InterviewFlashCards/AnswerViewController.swift]: `-=` and `+=` are deprecated in swift 3. Perhaps this is a good time to use the new convention?

File: InterviewFlashCards/AnswerViewController.swift [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+    
+    func showPaginationLabel() {
+        paginationLabel.text = "\(index+1)/\(flashCard.answerImages.count)"
+    }
+    
+    func handleLeftSwipe(gesture: UISwipeGestureRecognizer) {
+        if index < flashCard.answerImages.count-1 {
+            index += 1
+            showPaginationLabel()
+            answerImageView.image = flashCard.answerImages[index] as? UIImage
```
Comment (vhart) [inline InterviewFlashCards/AnswerViewController.swift]: is `answerImages` not already type `[UIImage]`?

Comment (charleshkang) [reply InterviewFlashCards/AnswerViewController.swift]: i had it as `answerImageView.image = flashCard.answerImages[index] `, but xcode suggested i cast it as `uiimage`

File: InterviewFlashCards/AnswerViewController.swift (hunk 3 above)
Comment (charleshkang) [reply InterviewFlashCards/AnswerViewController.swift]: what is it? i'm looking at: https://github.com/apple/swift-evolution/blob/master/proposals/0004-remove-pre-post-inc-decrement.md 

also this [link](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/BasicOperators.html) under `Compound Assignment Operators` is using `+=`

Comment (vhart) [reply InterviewFlashCards/AnswerViewController.swift]: You'd use index = index + 1

**Comment by vhart** [comment]
@mesbekmek @charleshkang I was wrong about += -=. What I was referring to was ++ and --.

**Comment by vhart** [comment]
@mesbekmek, the author of the proposal mentions that those operators are not much shorter than += -=, and the infinix and prefix operators have void return types being c operators, preventing inlining

**Comment by charleshkang** [comment]
so `+=` and `-=` are still valid in swift 3? with `index = index + 1` an alternative depending on preference?

**Comment by vhart** [comment]
@charleshkang yeah, += and -= are the recommended ways of incrementing now. as opposed to i++ and i--

**Comment by mesbekmek** [comment]
@charleshkang, @vhart out here making you change unnecessary things on purpose, I'm just looking out for you man 😆 😛

**Comment by vhart** [comment]
lol I clearly got fact checked.

**Comment by vhart** [comment]
@charleshkang If you could add the necessary changes and also rebase against master, we can get this merged in.

File: InterviewFlashCards/AnswerViewController.swift [hunk 5]
```diff
@@ -53,84 +48,10 @@ class AnswerViewController: UITableViewController {
...
-    
-    func handleRightSwipe(gesture: UISwipeGestureRecognizer) {
-        if index > 0 {
-            index+=1
-            showPaginationLabel()
-            answerImageView.image = flashCard.answerImages[index] as? UIImage
-        }
-    }
-    
-    func handlePinching(gesture: UIPinchGestureRecognizer) {
```
Comment (vhart) [inline InterviewFlashCards/AnswerViewController.swift]: pinching is for scaling images. Not sure if we want to remove it yet

**Comment by vhart** [comment]
@charleshkang marking this as inactive and closing. If you decide to fix the issues above, open another PR

**Comment by charleshkang** [comment]
@vhart so where are we back at regarding the swift 3 update, realm migration, and ui update? is the app back to the original ui?

**Pull request closed without merging**